docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md

```diff
@@ -252,7 +252,8 @@ You can add replicas to an existing primary cluster as needed. Refer to [Read re
 
 **Learn more**
```

Comment:
You can get rid of this Learn more and add links to "read replicas" and "follower reads" as the first two bullets under this section: https://deploy-preview-16228--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-cloud/cloud-basics/create-clusters-topology/#learn-more. Too much repetitive text. Or change the title from "Learn more" to something like "More information" for the read replicas section.